plugin/connection_control/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2016, 2022, Oracle and/or its affiliates.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License, version 2.0,
+# as published by the Free Software Foundation.
+#
+# This program is also distributed with certain software (including
+# but not limited to OpenSSL) that is licensed under separate terms,
+# as designated in a particular file or component or in included license
+# documentation.  The authors of MySQL hereby grant you an additional
+# permission to link the program and your derivative works with the
+# separately licensed software that they have included with MySQL.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License, version 2.0, for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  US
+MYSQL_ADD_PLUGIN(connection_control
+  connection_control.cc 
+  coordinator.cc
+  event_handler.cc
+  MODULE_ONLY  RECOMPILE_FOR_EMBEDDED
```

Comment:
these two keywords contradict each other

Reply:
Yes , but I found that these two cmake macro config writed in this file https://github.com/MariaDB/server/blob/10.11/plugin/server_audit/CMakeLists.txt . I want this plugin is a dynamic plugin and only can't be compiled as a dynamic module